tests/pipeline_tests.py

```diff
@@ -339,10 +339,15 @@ def test_outputs_exist(self) -> None:
         if self.data_reader_config.prompt_template_path:
             self.assertTrue(any("processed_prompts.jsonl" in str(file) for file in self.files))
         self.assertTrue(any("inference_result.jsonl" in str(file) for file in self.files))
+        self.verify_n_aggregators(self.eval_config)
+        
+    def verify_n_aggregators(self, eval_config) -> None:
+        eval_files = list(Path(self.eval_config.output_dir).rglob("*"))
+        self.eval_config = eval_config
         if self.eval_config.metric_config is not None:
-            self.assertTrue(any("metric_results.jsonl" in str(file) for file in self.files))
+            self.assertTrue(any("metric_results.jsonl" in str(file) for file in eval_files))
         n_aggregators = len(self.eval_config.aggregator_configs)
-        n_aggregator_files = len([file for file in self.files if "aggregator" in str(file)])
+        n_aggregator_files = len([file for file in eval_files if "aggregator" in str(file)])
         self.assertEqual(n_aggregators, n_aggregator_files)
 
 
```

Comment:
I think the AIME test got accidentally deleted?

Reply:
I thought it is on line 314?

Reply:
That is the pipeline. You need a unittest.TestCase type class to actually run the pipeline. Look at the other benchmark tests in this file.

Reply:
Updated the AIME test to reflect this